projects/examples/src/components/datagrid/datagrid-link.example.component.ts

```diff
@@ -3,136 +3,203 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
 import {
-    ButtonConfig,
-    ContextualButtonPosition,
+    ActionDisplayConfig,
+    ActionItem,
+    ActionType,
+    ContextualActionPosition,
+    DatagridComponent,
     GridColumn,
     GridDataFetchResult,
     GridSelectionType,
     GridState,
-    InactiveButtonDisplayMode,
     TextIcon,
 } from '@vcd/ui-components';
+import { ActionStyling } from '../../../../components/src/common/interfaces';
 
-interface Data {
+interface Record {
     value: string;
     paused: boolean;
 }
 
+interface Blah {
+    foo: string;
+    bar: string;
+}
+
+type HandlerData = Record[] | Blah;
+
 /**
  * Shows linked buttons on the top of the datagrid.
  * Has examples of both buttons that are global and contextual.
+ *
+ * Enable the delete button to hide it and disable the delete button to show it in disable mode. This is because the
+ * inactive display mode of a button can be set as following:
+ * Hidden - availability is false and disabled is false
```

Comment:
This should be a list

```ts
/** 
 * ...
 * Enable the delete button to hide it and disable the delete button to show it in disable mode. This is because the
 * inactive display mode of a button can be set as following:
 *  - Hidden - availability is false and disabled is false
 *  - Shown as Disabled - availability is false and disabled is true
 * Please refer to the delete action object configuration in the actions object list configuration.
```